tools/sdk-generation-pipeline/scripts/change-owner.sh

```diff
@@ -11,14 +11,12 @@ done
 if [ -d "${SPEC_REPO}" ]; then
   while true
```

Comment:
how exit the while loop?

Reply:
No need to exit while running. You can check the entrypoint.sh, and you will find it runs in backend.